### ISSUE TITLE
Limited day is clickable

### ISSUE THREAD

**Issue by antonkomarev** [hilongjw/vue-datepicker#88]
![selection_227](https://cloud.githubusercontent.com/assets/1849174/22574443/eb2d4c36-e9bf-11e6-91f6-0563a09e45a6.png)

Dates are limited from 4th of february till 23rd. Dates highlighted with green marker are unclickable. Dates marked with red - are broken: could be selected!

Using in Vue.js v1

*Note: Days from previous month are all unclickable.*

**Comment by n8-dev** [comment]
+1

IMO red should just take you to the next month, not actually be selectable.

**Comment by antonkomarev** [comment]
@n8-sd Yes, but if the are unavailable to be selected - they should be unclickable.

**Comment by vwakeup** [comment]
find code
var passiveDaysAtFinal = 42 - days.length;
      for (var _i2 = 1; _i2 <= passiveDaysAtFinal; _i2++) {
        var _passiveDay = {
          value: _i2,
          inMonth: false,
          action: 'next',
          unavailable: false,
          checked: false,
          moment: (0, _moment2.default)(currentMoment).add(1, 'months').date(_i2)
        };

then move up before  
if (this.limit.length > 0) {

**Comment by hu3402379** [comment]
find code and change unavailable from false to true. 

for (let i = 0; i < firstDay - (this.option.SundayFirst ? 0 : 1); i++) {
                let passiveDay = {
                    value: previousMonth.daysInMonth() - (i),
                    inMonth: false,
                    action: 'previous',
                    **unavailable: true,**
                    checked: false,
                    moment: moment(currentMoment).date(1).subtract(i + 1, 'days')
                }
                days.unshift(passiveDay)
            }

let passiveDaysAtFinal = 42 - days.length
            for (let i = 1; i <= passiveDaysAtFinal; i++) {
                let passiveDay = {
                    value: i,
                    inMonth: false,
                    action: 'next',
                    **unavailable: true,**
                    checked: false,
                    moment: moment(currentMoment).add(1, 'months').date(i)
                }
                days.push(passiveDay)
            }

**Comment by JStrebeyko** [comment]
thank you! :)
